### DISCUSSION TITLE
Design credit on personal project

### DISCUSSION THREAD

**Discussion by hicallmeal** [Authenticator-Extension/Authenticator#826] [General]
Hello, 

I'm developing my own OTP extension and I've concluded that the box design that you use is most suitable. Given I'd like to one day release my project to extension stores and the designs look quite similar, would you like me to credit you in some fashion? 

This would be the only similarity, otherwise my project is mostly different functionality wise.
Though of course it will share the same principles and some ideas - privacy, etc.

Thanks,

al

**Comment by Sneezry** [comment]
Hi @hicallmeal , thanks for reaching out. Authenticator is an open source project, and you are free to use its source code and UI design. However, please do not use [the same icon](https://raw.githubusercontent.com/Authenticator-Extension/Authenticator/dev/images/icon.svg) to prevent misunderstanding.

**Reply by hicallmeal** [reply]
Thanks for the prompt response - I'll be sure to follow that, as well as respecting other factors such as the name, etc. 
I believe in giving credit where credit is due, so let me know in the future if you feel you should be credited.

Al

**Reply by Sneezry** [reply]
Authenticator is released under MIT license. MIT requires to keep [the copyright information](https://raw.githubusercontent.com/Authenticator-Extension/Authenticator/dev/LICENSE). To my personal mind, this is a better-to-have requirement, but not a must-have requirement. So, it's up to you :)
